docs/3rd_party/numpy/snippets/find_duplicate_records.py

```diff
@@ -0,0 +1,11 @@
+import numpy as np
+
+
+def find_duplicate_records(arr: np.ndarray) -> np.ndarray:
+    """Returns duplicate records from a numpy array
+    Example:
+        >>> arr = np.array([1, 2, 3, 2, 4, 5, 6, 3, 7, 8, 1, 9, 2, 3])
+    """
```

Comment:
Nit and small tip from @fabio:
```
"""
Returns duplicate records from a numpy array

Usage:
    >>> arr = np.array([1, 2, 3, 2, 4, 5, 6, 3, 7, 8, 1, 9, 2, 3])
"""
```

Note the leading EOL on multi-line docstrings and the preference for "Usage" over "Example" now.

----

Not communicated properly everywhere, but here's some [context](https://robocorptechnologies.slack.com/archives/C04SAQERGCF/p1706624644076339?thread_ts=1706622474.642509&cid=C04SAQERGCF).